ssz/src/main/java/net/consensys/cava/ssz/SSZ.java

```diff
@@ -487,6 +487,23 @@ static void encodeBytesListTo(Bytes[] elements, Consumer<Bytes> appender) {
     }
   }
 
+  static void encodeBytesListTo(List<Bytes> elements, Consumer<Bytes> appender) {
```

Comment:
Also add `Bytes encodeBytesList(List<Bytes> elements)`?

Reply:
A subsequent commit removes encodeBytesList. It was only used from a test within BytesSSZWriterTest seemingly as a backdoor to test the default visibility method `encodeBytesListTo`. I think it would be preferred to test directly using the `writeBytesList` method in `SSZWriter` so that's what I did.

Reply:
It wasn't a backdoor, it was part of the SSZ API. All the encoding methods have static public `SSZ.encodeXXX` methods.

The package-private visible `encodeXXXListTo(...)` methods do not form part of the visible API, and only exist in the SSZ class to avoid duplication between the implementation of `SSZ.encodeXXXList(...)` and `SSZWriter#encodeXXXList(...)`.

Reply:
Yeah - there could be some more tests, for sure